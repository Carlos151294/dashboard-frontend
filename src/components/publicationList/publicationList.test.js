import React from 'react';
import { shallow } from 'enzyme';
import PublicationList from './publicationList.component';

it('Excepect to render PublicationList component', () => {
    expect(shallow(<PublicationList />)).toMatchSnapshot();
});