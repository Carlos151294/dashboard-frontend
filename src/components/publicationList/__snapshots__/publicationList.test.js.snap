// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Excepect to render PublicationList component 1`] = `ShallowWrapper {}`;
